Parsedown.php

```diff
@@ -1217,7 +1217,7 @@ protected function inlineLink($Excerpt)
             return;
         }
 
-        if (preg_match('/^[(]((?:[^ ()]++|[(][^ )]+[)])++)(?:[ ]+("[^"]*"|\'[^\']*\'))?[)]/', $remainder, $matches))
+        if (preg_match('/^[(]\s*+((?:[^ ()]++|[(][^ )]+[)])++)(?:[ ]+("[^"]*"|\'[^\']*\'))?\s*[)]/', $remainder, $matches))
```

Comment:
what's the role of the `+` in 
![image](https://cloud.githubusercontent.com/assets/184170/22177111/a7da6f48-e020-11e6-83e4-df1580ed1ea1.png)